storage/rocksdb/ha_rocksdb.cc

```diff
@@ -3477,8 +3486,8 @@ static void rocksdb_update_table_stats(
     my_core::filename_to_tablename(tablename.c_str(), tablename_sys,
                                    sizeof(tablename_sys));
     (*cb)(dbname_sys, tablename_sys, is_partition, &io_perf_read, &io_perf,
-          &io_perf, &io_perf, &io_perf, &page_stats, &comp_stats, 0, 0,
-          rocksdb_hton_name);
+          &io_perf, &io_perf, &io_perf, &page_stats, &comp_stats, 0,
+          lock_wait_timeout_stats, deadlock_stats, rocksdb_hton_name);
```

Comment:
it seems to me that if we are going to add a new `deadlock_stats` field in the table statistics, we may as well also add a corresponding `rocksdb_row_lock_waits` and `rocksdb_row_lock_wait_timeout`.   I feel that users may be confused when they see `innodb_row_lock_wait_timeout` increasing when using MyRocks and not InnoDB.  Or we could consider renaming the variable to something more general. e.g. (row_lock_waits/row_lock_wait_timeout). Any reason we haven't taken this approach?  cc @lth